templates/education/course-cancellation.njk

```diff
@@ -0,0 +1,37 @@
+<div class="panel panel-danger">
```

Comment:
where is this content going? seems to be a duplication of the markdown content and is actually not reflected anywhere on the website that I can find.